src/screens/Login.js

```diff
@@ -1,51 +1,79 @@
 import React, { Component } from 'react';
 import { auth } from 'firebase';
 import {
-  StyleSheet, TextInput, View, Text, StatusBar, KeyboardAvoidingView,
+  StyleSheet, TextInput, View, Text, KeyboardAvoidingView, ImageBackground, Image, TouchableOpacity,
 } from 'react-native';
 import PropTypes from 'prop-types';
+import { LinearGradient } from 'expo';
 
-import OpacityButton from '../components/OpacityButton';
 import Error from './Error';
+import buttonStyles from '../styles/Button';
 
 const localColors = {
   lightWhite: '#D8D8D8',
+  white: '#fff',
+  black: '#000',
+  red: '#E03E18',
+  redGradient1: '#E93D13',
+  redGradient2: '#DF3C16',
 };
 
 const styles = StyleSheet.create({
-  controlPanel: {
-    flexDirection: 'column',
-  },
-  buttonsLayout: {
-    flexDirection: 'row',
-    alignItems: 'center',
-    justifyContent: 'center',
-  },
-  inputStyle: {
-    backgroundColor: localColors.lightWhite,
-    paddingHorizontal: 10,
-    paddingVertical: 10,
-    width: '60%',
-    margin: 15,
+  backgroundImage: {
+    width: '100%',
+    height: '100%',
   },
-  viewStyle: {
+  view: {
     display: 'flex',
     alignItems: 'center',
+  },
+  keyboardAvoidingView: {
     paddingTop: '10%',
     width: '100%',
   },
-  backgroundStyle: {
+  logoView: {
```

Comment:
I think that this kind of styles should be in a seperate file so that other coponents could use it? We can leave it here as now nothing uses logo, but keep that in mind not to repeat the code. (because the logo will appear in other places as well, right?)